airflow/contrib/hooks/bigquery_hook.py

```diff
@@ -207,6 +207,88 @@ def __init__(self, service, project_id, use_legacy_sql=True):
         self.use_legacy_sql = use_legacy_sql
         self.running_job_id = None
 
+    def create_empty_table(self,
+                           project_id,
+                           dataset_id,
+                           table_id,
+                           schema_fields=None,
+                           time_partitioning={}
+                           ):
+        """
+        Creates a new, empty table in the dataset.
+
+        :param project_id: The project to create the table into.
+        :type project_id: str
+        :param dataset_id: The dataset to create the table into.
+        :type dataset_id: str
+        :param table_id: The Name of the table to be created.
+        :type table_id: str
+        :param schema_fields: The schema field list as defined here: ::
+            https://cloud.google.com/bigquery/docs/reference/rest/v2/tables#resource
```

Comment:
could you also provide an example here for schema_fields?

Reply:
Example Added